Source/Core/DiscIO/NANDImporter.cpp

```diff
@@ -130,52 +130,46 @@ std::string NANDImporter::GetPath(const NANDFSTEntry& entry, const std::string&
 
 std::string NANDImporter::FormatDebugString(const NANDFSTEntry& entry)
 {
-  return fmt::format("{:12.12} {:#010b} {:#04x} {:#06x} {:#06x} {:#010x} {:#010x} {:#06x} {:#010x}",
-                     entry.name, entry.mode, entry.attr, entry.sub, entry.sib, entry.size,
-                     entry.uid, entry.gid, entry.x3);
+  return fmt::format("{:#010b} {:#04x} {:#06x} {:#06x} {:#010x} {:#010x} {:#06x} {:#010x}",
+                     entry.mode, entry.attr, entry.sub, entry.sib, entry.size, entry.uid, entry.gid,
+                     entry.x3);
 }
 
 void NANDImporter::ProcessEntry(u16 entry_number, const std::string& parent_path)
 {
   NANDFSTEntry entry;
-  memcpy(&entry, &m_nand[m_nand_fst_offset + sizeof(NANDFSTEntry) * entry_number],
-         sizeof(NANDFSTEntry));
-
-  if (entry.sib != 0xffff)
-    ProcessEntry(entry.sib, parent_path);
-
-  Type type = static_cast<Type>(entry.mode & 3);
-  if (type == Type::File)
-    ProcessFile(entry, parent_path);
-  else if (type == Type::Directory)
-    ProcessDirectory(entry, parent_path);
-  else
-    ERROR_LOG_FMT(DISCIO, "Ignoring unknown entry type {}", static_cast<int>(type));
+  while (entry_number != 0xffff)
+  {
+    memcpy(&entry, &m_nand[m_nand_fst_offset + sizeof(NANDFSTEntry) * entry_number],
+           sizeof(NANDFSTEntry));
+
+    const std::string path = GetPath(entry, parent_path);
+    INFO_LOG_FMT(DISCIO, "{} {}", FormatDebugString(entry), path);
```

Comment:
This seems like leftover debug logging (introduced in "NANDImporter: Reduce recursion in `ProcessEntry`").  Or at least, I think it would benefit from additional context as to what it's logging in the message.